src/Microsoft.AspNetCore.OData/Extensions/ODataServiceCollectionExtensions.cs

```diff
@@ -139,5 +139,31 @@ public static IServiceCollection AddODataQueryFilter(this IServiceCollection ser
             services.TryAddEnumerable(ServiceDescriptor.Singleton<IFilterProvider>(new QueryFilterProvider(queryFilter)));
             return services;
         }
+
+        /// <summary>
+        /// Enables support for case insensitive model binding.
+        /// </summary>
+        /// <param name="services">The services collection.</param>
+        public static IServiceCollection EnableODataCaseInsensitiveModelBinding(this IServiceCollection services)
```

Comment:
In the asp.net core version, please add the extensions to "IODataBuilder".

So, customer should call "AddOData().UseCaseInsensitiveBinding()"

Reply:
#Resolved